tests/benchmarks/test_registry.py

```diff
@@ -196,3 +197,20 @@ def test_launch_benchmark():
     benchmark = BenchmarkRegistry.launch_benchmark(context)
     assert (benchmark)
     assert (benchmark.return_code == ReturnCode.INVALID_ARGUMENT)
+
+
+def test_get_all_benchmark_predefine_settings():
+    """Test interface BenchmarkRegistry.get_all_benchmark_predefine_settings()."""
+    benchmark_params = BenchmarkRegistry.get_all_benchmark_predefine_settings()
+
+    # Choose benchmark 'pytorch-sharding-matmul' for testing.
+    benchmark_name = 'pytorch-sharding-matmul'
+    assert (benchmark_name in benchmark_params)
+    assert (benchmark_params[benchmark_name]['run_count'] == 1)
```

Comment:
`assertDictEqual` can be used for two dicts in `unittest.TestCase`

Reply:
Thanks for suggestion.